Tests/LibWeb/Text/input/wpt-import/url/url-constructor.any.html

```diff
@@ -0,0 +1,15 @@
+<!doctype html>
+<meta charset=utf-8>
+<meta name="timeout" content="long">
+<script>
+self.GLOBAL = {
+  isWindow: function() { return true; },
+  isWorker: function() { return false; },
+  isShadowRealm: function() { return false; },
+};
+</script>
+<script src="../resources/testharness.js"></script>
+<script src="../resources/testharnessreport.js"></script>
+<script src="../common/subset-tests-by-key.js"></script>
+<div id=log></div>
+<script src="../url/url-constructor.any.js"></script>
```

Comment:
I had tried to import the test from `http://wpt.live/url/url-constructor.any.worker.html?exclude=(file|javascript|mailto)` but it failed (I presume it was due to the usage of Worker here int he original).
Are the docs incomplete or did I miss anything?

Reply:
I had to manually download the resources JSON file since the importer is not able to currently detect it as a dependency. That is what I guess you ran into

The URL I used was http://wpt.live/url/url-constructor.html